server/src/main/java/org/opensearch/action/main/MainResponse.java

```diff
@@ -149,6 +149,7 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             .field("minimum_wire_compatibility_version", version.minimumCompatibilityVersion().toString())
             .field("minimum_index_compatibility_version", version.minimumIndexCompatibilityVersion().toString())
             .endObject();
+        builder.field("tagline", "The OpenSearch Project: https://opensearch.org/");
```

Comment:
I feel the value should be turned into constant.

Reply:
I agree with it... I can take some time in finding a good place for the constant.

Reply:
Made the tagline become a constant in this class. Thank you!